src/components/CreateHospital.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { Mutation } from 'react-apollo';
+import gql from 'graphql-tag';
+import { ALL_HOSPITALS_QUERY } from './HospitalTable';
+
+const ADD_HOSPITAL_MUTATION = gql`
+    mutation PostMutation($name: String!) {
+        addHospital(name: $name){
+            id
+            name
+            createdAt
+            updatedAt
+        }
+    }
+`;
+
+class CreateHospital extends Component {
```

Comment:
I think it might be better practice if we stuck to functional components instead of class components. App is already a functional component, and we could also use the new React hooks. Apollo client has some nice support for React hooks - they wrote their own `useQuery`, and other hooks for CRUD operations.

Reply:
yep, we should definitely move towards hooks and functional components - this is probably one of the first changes i'd like to make to what's here right now